subprojects/base-services/src/main/java/org/gradle/internal/classloader/ClasspathUtil.java

```diff
@@ -38,7 +38,7 @@
 public class ClasspathUtil {
     public static void addUrl(URLClassLoader classLoader, Iterable<URL> classpathElements) {
         try {
-            Set<URI> original = new HashSet<URI>();
+            Set<URI> original = new HashSet<>();
```

Comment:
sorry that actually was my Eclipse Java 8 auto Save Action, not intentional proposed changed; removed now